systems/stats.md

```diff
@@ -1,8 +1,51 @@
 
-## Defence
+# Attributes
+These attributes intrinsic to your character. They represent the base capabilities of your body and mind. These will only change rarely throughout a characters journey.
+
+These are all recorded as a numeric modifier. An attribute of `0` represents an unexceptional value that a non adventuring human citizen might have.

```

Comment:
Specifying human in a potentially non human world. Maybe just person or character